nextflow.config

```diff
@@ -5,3 +5,9 @@ process {
     '-A b2015110 --qos=short -p node'
   }
 }
+
+params {
+    programs {
```

Comment:
@viklund How does this work ... is "programs" something that becomes a flag on the commandline, or?